qa/TL0_python-self-test-core/test_body.sh

```diff
@@ -40,14 +40,23 @@ test_type_annotations() {
 
 test_pytorch() {
     ${python_invoke_test} --attr '!slow,pytorch' test_dali_variable_batch_size.py
-    ${python_new_invoke_test} -A '!slow' checkpointing.test_dali_checkpointing_fw_iterators.TestPytorch
-    ${python_new_invoke_test} -A '!slow' checkpointing.test_dali_checkpointing_fw_iterators.TestPytorchRagged
     ${python_new_invoke_test} -A 'pytorch' -s type_annotations
+    if [ -z "$DALI_ENABLE_SANITIZERS" ]; then
+        ${python_new_invoke_test} -A '!slow' checkpointing.test_dali_checkpointing_fw_iterators.TestPytorch
+        ${python_new_invoke_test} -A '!slow' checkpointing.test_dali_checkpointing_fw_iterators.TestPytorchRagged
+    fi
 }
 
 test_checkpointing() {
-    ${python_new_invoke_test} -A '!slow,!pytorch,!mxnet,!cupy,!numba' checkpointing.test_dali_checkpointing
-    ${python_new_invoke_test} -A '!slow,!pytorch,!mxnet,!cupy,!numba' checkpointing.test_dali_stateless_operators
+    if [ -z "$DALI_ENABLE_SANITIZERS" ]; then
+        ${python_new_invoke_test} -A '!slow,!pytorch,!mxnet,!cupy,!numba' checkpointing.test_dali_checkpointing
+        ${python_new_invoke_test} -A '!slow,!pytorch,!mxnet,!cupy,!numba' checkpointing.test_dali_stateless_operators
+    else
+        ${python_new_invoke_test} -A '!slow,!pytorch,!mxnet,!cupy,!numba,!sanitizer_skip' checkpointing.test_dali_checkpointing
+
+        # External source tests are slow and Python-side mostly, but let's run just one of them
+        ${python_new_invoke_test} -A '!slow,!pytorch,!mxnet,!cupy,!numba,!san_skip' checkpointing.test_dali_checkpointing.test_external_source_checkpointing:1
```

Comment:
Do we use `san_skip` anywhere?

Reply:
Oops! It's a mistake (or two mistakes): 1) we don't use san_skip, we use sanitizer_skip 2) here we *don't* want to skip the test. Thanks!

Reply:
Removed